mod-source-record-manager-server/src/main/resources/templates/db_scripts/schema.json

```diff
@@ -168,6 +168,11 @@
       "snippet": "DO $$ BEGIN BEGIN create type rule_type AS ENUM ('MARC_BIB', 'MARC_HOLDING'); EXCEPTION WHEN duplicate_object THEN NULL; END; END $$; ALTER TABLE mapping_rules ADD COLUMN IF NOT EXISTS record_type rule_type NOT NULL DEFAULT 'MARC_BIB';",
       "fromModuleVersion": "mod-source-record-manager-3.2.0"
     },
+    {
+      "run": "after",
+      "snippet": "ALTER TABLE mapping_rules ADD CONSTRAINT unq_record_type UNIQUE (record_type);",
+      "fromModuleVersion": "mod-source-record-manager-3.2.6"
```

Comment:
I'm not very familiar with schema updating in RMB. Can you approve that I set the right version?
This change will be cherry-picked to kiwi release branch

Reply:
hi @psmagin, yes the fromModuleVersion is correct